example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -111,6 +111,7 @@ describe('The S3 Ingest Granules workflow', () => {
   const collectionModel = new Collection();
   process.env.ProvidersTable = `${config.stackName}-ProvidersTable`;
   const providerModel = new Provider();
+  process.env.RulesTable = `${config.stackName}-RulesTable`;
```

Comment:
Try removing this

Reply:
I've been running just this test in a loop and it seems happy enough without it.